qiskit/providers/backend_compat.py

```diff
@@ -122,7 +126,7 @@ def convert_to_target(
         target.add_instruction(Measure(), measure_props)
     # Parse from configuration because properties doesn't exist
     else:
-        target = Target(num_qubits=configuration.n_qubits)
+        target = Target(num_qubits=configuration.n_qubits, meas_map=configuration.meas_map)
```

Comment:
The type hint of `configuration` is `BackendCOnfiguration` now. Currently tests doesn't pass because an attribute `meas_map` doesn't exist in `BackendCOnfiguration`.